### PR TITLE
Feature: add logo lockup

### PR DESCRIPTION
**Pull request by ScottAntonacAP**
## Summary of Changes

- Add the logo lockup as a custom view
- Add the logo lockup as an option for the price breakdown

Screen Shot 2022-04-07 at 12 13 17

## Submission Checklist

- [x] Documentation is changed or added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -324,6 +332,20 @@ Setting `introText` is optional, will default to `OR` and must be of type `After
 Can be any of `OR`, `OR_TITLE`, `MAKE`, `MAKE_TITLE`, `PAY`, `PAY_TITLE`, `IN`, `IN_TITLE`, `PAY_IN`, `PAY_IN_TITLE` or `EMPTY` (no intro text).
 Intro text will be rendered lowercase unless using an option suffixed with `_TITLE` in which case title case will be rendered.
 
+##### Logo Type
+Setting `logoType` is optional, will default to `BADGE` and must be of type `AfterpayLogoType`.
+
+Can be either of `BADGE` of `LOCKUP`.
```
Comment (benfterpay) [inline README.md:338]:
```suggestion
Can be either of `BADGE` or `LOCKUP`.
```

File: README.md [hunk 2]
```diff
@@ -324,6 +332,20 @@ Setting `introText` is optional, will default to `OR` and must be of type `After
...
+When setting color scheme on logo type of `LOCKUP`, only the foreground color will be applied. (See example)
+
+```kotlin
+val afterpayBreakdown = view.findViewById<AfterpayPriceBreakdown>(R.id.afterpayPriceBreakdown)
+afterpayBreakdown.logoType = AfterpayLogoType.LOCKUP
+afterpayBreakdown.colorScheme = AfterpayColorScheme.MINT_ON_BLACK
+```
+
+Given the above, the price breakdown will contain the lockup logo and will be be of color mint.
```
Comment (benfterpay) [inline README.md:347]:
```suggestion
Given the above, the price breakdown will contain the lockup logo and will be of color mint.
```

File: README.md [hunk 3]
```diff
@@ -432,6 +454,9 @@ This project is licensed under the terms of the Apache 2.0 license. See the [LIC
 [badge-mint-on-black]: images/badge_mint_on_black.png
 [badge-black-on-white]: images/badge_black_on_white.png
 [badge-white-on-black]: images/badge_white_on_black.png
+[lockup-black]: images/lockup_black.png
+[lockup-white]: images/lockup_white.png
+[lockup-mint]: images/lockup_mint.png
```
Comment (benfterpay) [inline README.md:459]: Please update the example PNG for the mint lockup to have a dark background. Both the white and the mint lockups are intended to be used only on a dark background. Refer to https://www.afterpay.com/en-US/retailer-resources

**Review by benfterpay — CHANGES_REQUESTED** [review]
This looks really tidy. Just suggesting some minor tweaks. Thank you.

File: README.md (hunk 3 above)
Comment (ScottAntonacAP) [reply README.md:459]: Thanks for bringing this to my attention - addressed in 87545e1

**Review by benfterpay — APPROVED** [review]

**Pull request merged**